packages/editor/src/components/rich-text/index.native.js

```diff
@@ -69,6 +70,10 @@ const gutenbergFormatNamesToAztec = {
 	'core/strikethrough': 'strikethrough',
 };
 
+const headingTags = [
+	'h1', 'h2', 'h3', 'h4', 'h5', 'h6',
```

Comment:
Adding headings related constants and logic seems out of place inside the RichText component, adding a "cyclic" like relationship between the RichText and the heading block.

What do you think instead, to add a flag as a prop to RichText which we will use to bypass the setting of the `html` variable to empty later in the render method?

Reply:
ah yes! good point!

Reply:
@hypest fixed, let me know if you have further comments.